protos/transponder/transponder.proto

```diff
@@ -53,19 +53,26 @@ enum AdsbEmitterType {
     ADSB_EMITTER_TYPE_POINT_OBSTACLE = 19; // Point obstacle emitter.
 }
 
+enum AdsbAltitudeType {
+    ADSB_ALTITUDE_TYPE_PRESSURE_QNH = 0; // Altitude reported from a Baro source using QNH reference
+    ADSB_ALTITUDE_TYPE_GEOMETRIC = 1; // Altitude reported from a GNSS source
+}
+
+
 // ADSB Vehicle type.
 message AdsbVehicle {
     uint32 icao_address = 1; // ICAO (International Civil Aviation Organization) unique worldwide identifier
     double latitude_deg = 2; // Latitude in degrees (range: -90 to +90)
     double longitude_deg = 3; // Longitude in degrees (range: -180 to +180).
-    float absolute_altitude_m = 5; // Altitude AMSL (above mean sea level) in metres
+    float altitude_m = 5; // Altitude ASL (above sea level) in metres
```

Comment:
Do we really need to rename that field? That will break backward compatibility.

Keeping the old name and using a default value for `altitude_type` in the C++ implementation should stay backward compatible, which would be nicer, I think.